expression/builtin_miscellaneous_vec.go

```diff
@@ -159,11 +160,11 @@ func (b *builtinNameConstStringSig) vecEvalString(input *chunk.Chunk, result *ch
 }
 
 func (b *builtinDecimalAnyValueSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinDecimalAnyValueSig) vecEvalDecimal(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	return b.args[1].VecEvalDecimal(b.ctx, input, result)
```

Comment:
Is this modification related to this PR, or should we add a test case for this function signature?